src/3D/Camera.h

```diff
@@ -47,6 +55,8 @@ class Camera
 
 	[[nodiscard]] glm::vec3 GetPosition() const { return _position; }
 	[[nodiscard]] glm::vec3 GetRotation() const { return glm::degrees(_rotation); }
+	[[nodiscard]] glm::vec3 GetMovementSpeed() const { return _movementSpeed; }
```

Comment:
Speed + direction become a velocity.
```
GetVelocity
_velocity
```

Reply:
Good catch. Thanks for spotting this. I wasn't using `_velocity` at all